specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/definitions.json

```diff
@@ -1939,6 +1939,27 @@
       ],
       "description": "Authorization header information."
     },
+    "BackendManagedIdentityCredentials": {
+      "type": "object",
+      "properties": {
+        "resource": {
+          "type": "string",
+          "description": "The application ID of the target web API (secured resource) in Microsoft Entra ID.",
+          "minLength": 1,
+          "maxLength": 300
+        },
+        "clientId": {
+          "type": "string",
+          "description": "The client ID of the user-assigned identity in Microsoft Entra ID.",
+          "minLength": 1,
+          "maxLength": 300
+        }
+      },
+      "required": [
+        "resource"
```

Comment:
>resource [](http://example.com/codeflow?start=9&length=8)

because this is required it impacts the PATCH operations using it.

Reply:
@razvanbadea-msft Thanks for reviewing. Trying to understand what to do to address this?
- To add an example for PATCH? Just added that.
- Should not set this property to required?
- Other?
Not sure I understand your implication.

Reply:
Gentle ping @razvanbadea-msft